docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
+Für die Entwickler wurden auch die Pull Request Tests um Typescript Support erweitert.
+\subsubsection{Retrospektive}
+Besonders gut gefiel dem Team das Teamklima, vor allem des ehrliche Miteinander.
+Trotz Komplikationen und Probleme konnte am Ende dem Kunden ein schönes Product Increment geliefert werden.
+Pull Requests und Tickets wurden besser beschrieben, sodass Probleme besser verständlicher waren.
+Vor allem durch das interne Team Review wurde die Zusammenarbeit weiter gebessert.
+Das Team konnte weiterhin das Wissen mit Vue ausbauen, vor allem zu Reaktivität, Events und Kommunikation zwischen Komponenten.
+Stark bemängelt wurde der Scope des Meilensteins.
+Das Zeitmanagement war an dieser Stelle nicht gut, das Team hatte sich zu viel vorgenommen und am Ende zu wenig Zeit.
+Das Team stimmte hier überein, dass eine bessere Planung sowie Abstimmung für die nächsten Meilensteine notwendig wäre.
+Auch sollte eine frühere interne Deadline gesetzt werden, damit nicht kurz vor dem Sprintende noch viele Fehler auftreten und behoben werden müssen.
+Um insgesamt auch die Arbeit kontinuierlicher zu gestalten, sollten Aufgaben in kleinere Probleme aufgeteilt werden.
+
+\subsection{Meilenstein 5}
+Der Meilenstein fünf vom 14.01.2021 bis zum 28.01.2021 sollte dafür dienen, die Entwicklung am Produkt weitgehend zu beenden, sodass sich das Team darauf auf den Bericht fokussieren konnte.
+\subsubsection{Ziel}
+Geplant war, die letzten Anwendungszenarien final umzusetzen, sodass die letzten zwei Meilensteine höchstens zum Beheben kleinerer Fehler benötigt werden würden.
+Somit war das Ziel die Umsetzung der Einstellungen für die \textit{Leek Box} selbst als auch der einzelnen Tags.
+Die User Interface Elemente sollte, sofern noch nicht geschehen, ein einheitliches Design erhalten.
+Auch sollte die Dokumentation zur Bedienung der \textit{Leek Box} fertig gestellt sein und die Arbeit für den Bericht beginnen.
+\subsubsection{Probleme}
+Wie beim vorherigen Meilenstein hat auch hier das Zeitmanagement nicht komplett funktioniert.
+Gegen Ende fehlte es an Zeit, um neue umgesetzte Tickets auf Fehler zu testen und entsprechend fehlerfrei dem Kunden zu präsentieren.
+\subsubsection{Lösungen}
+Die einzige mögliche Lösung für das Team zu diesem Problem war am Ende dem Kunden weniger neue Features vorzustellen als geplant.
+\subsubsection{Product Increment}
+Die \textit{Leek Box} Einstellungen sind implementiert worden, über die der Nutzer den Wiedergabelautsprecher auswählen, die jeweilige Leek Box verwalten und sich auch ausloggen kann.
+Auch wurde ein Setup Guide angefertigt, sodass ein Nutzer selbstständig eine eigene \textit{Leek Box} einrichten könnte.
+Bei dem Design wurden alle Unstimmigkeiten noch beseitigt.
+\subsubsection{Retrospektive}
+Insgesamt hat das Team sich an dieser Stelle vor allem über das sehr positive Feedback des Kunden gefreut.
+Auch war die Produktivität des Teams sehr gut.
+Das Team konnte auch weitere Vue Funktionen kennenlernen.
```

Comment:
```suggestion
Das Team konnte außerdem weitere Vue Funktionen kennenlernen.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee